docs/source/specifications/driver-override.rst

```diff
@@ -0,0 +1,54 @@
+.. _DriverOverride:
+
+Driver Override Specification
+==============================
+This document describes the specification for the global override feature.
+By default, every user is allowed write access to the devices by the master driver. The override feature will allow the user (for example, building administrator) to override this default behavior and enable the user to lock the write access on the devices for specified duration of time .
```

Comment:
If single time period it should be for "a" specified duration.  Is there a way to "lock it indefinately?"